build/UserALEWebExtension/options.js

```diff
@@ -1142,13 +1142,17 @@ function setConfig() {
   if (config.userId && password) {
     config.authHeader = "Basic " + btoa("".concat(config.userId, ":").concat(password));
   }
-  browser.storage.local.set({
-    useraleConfig: config
-  }, function () {
+  var payload = {
+    useraleConfig: config,
+    pluginConfig: {
+      urlWhitelist: document.getElementById("filter").value
+    }
+  };
+  browser.storage.local.set(payload, function () {
     options(config);
     browser.runtime.sendMessage({
       type: CONFIG_CHANGE,
-      payload: config
+      payload: payload
```

Comment:
FYI: JS will automatically infer the key if your value of the same name.

So `{type: CONFIG_CHANGE, payload }` is perfectly valid. Saves you a few characters of typing.